### PR TITLE
[fix] remove nul.d.ts file

### LINKED ISSUE #55700: error: invalid path 'types/barnard59-base/nul.d.ts' when working with repo on Windows OS
- [x] Check the [Infrastructure status updates](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/44317) thread for known issues.
- [x] Leave a message on the [DefinitelyTyped channel in the TypeScript Community Discord](https://discord.gg/typescript).

This is a git trivia, there is a new DT package which contains a file for a module, which defines `nul` type,
resulting `nul.d.ts` file causes error now on Windows OS when using git:
```bash
error: invalid path 'types/barnard59-base/nul.d.ts'
```
as the `nul` is (still) not allowed filename on Windows OS.

### PR DESCRIPTION
**Pull request by peterblazejewicz**
- remove file
- move definition to index file

Temporary fix for Windows OS, for git failing to work
properly with reserved filename in usual scenarios.

Closes #55700

- [x] Use a meaningful title for the pull request. Include the name of the package modified.

### CODE REVIEW

File: types/barnard59-base/index.d.ts [hunk 1]
```diff
@@ -3,6 +3,10 @@
 // Definitions by: Tomasz Pluskiewicz <https://github.com/tpluscode>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+/// <reference types="node" />
+
+import { Writable } from 'stream';
```
Comment (peterblazejewicz) [inline types/barnard59-base/index.d.ts:8]:
```suggestion
import { Writable } from 'readable-stream';
```

File: types/barnard59-base/barnard59-base-tests.ts [hunk 2]
```diff
@@ -56,6 +62,7 @@ function testFlatten() {
 }
 
 function testForEach() {
+    // prettier-ignore
```
Comment (tpluscode) [inline types/barnard59-base/barnard59-base-tests.ts:65]: Why are the `prettier-ignore` necessary? Looks like there will be many across the types packages

**Review by tpluscode — COMMENTED** [review]
Is this to reflect some new formatting rules? Would be nice for them to be either part of `npm test` (if not already) or, better yet, enforced with a precommit hook such as `husky+lint-staged` combination I use in my repos

**Review by tpluscode — APPROVED** [review]
Let's not block this. Looks Good

**Review by orta — APPROVED** [review]

**Comment by tpluscode** [comment]
Ready to merge

**Comment by peterblazejewicz** [comment]
thx!

**Comment by peterblazejewicz** [comment]
fixed (win 11)

**Pull request merged**